starters/oasp4j-batch-starter/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.oasp.java.dev</groupId>
+    <artifactId>oasp4j-starters</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+  <groupId>io.oasp.java.starters</groupId>
+  <artifactId>oasp4j-batch-starter</artifactId>
+  <version>${oasp4j.version}</version>
+  <packaging>jar</packaging>
+  <name>${project.artifactId}</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>${oasp.modules.groupId}</groupId>
+      <artifactId>oasp4j-batch</artifactId>
+      <version>${oasp4j.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>${oasp.modules.groupId}</groupId>
+      <artifactId>oasp4j-test</artifactId>
+      <version>${oasp4j.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-autoconfigure</artifactId>
+      <version>${spring.boot.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.batch</groupId>
+      <artifactId>spring-batch-test</artifactId>
```

Comment:
This is IMHO wrong. Test dependencies have to be added to the test starter and need to be added to the project with `<scope>test</scope>`.